exercises/simple-linked-list/src/example/java/SimpleLinkedList.java

```diff
@@ -0,0 +1,79 @@
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
+/**
+ *
+ * @author matt
+ */
+public class SimpleLinkedList {
+
+    class Element {
+
+        int value;
+        Element next;
+
+        Element(int value) {
+            this.value = value;
+        }
+    }
+
+    private Element head;
+    private int size = 0;
+
+    public SimpleLinkedList() {
+    }
+
+    public SimpleLinkedList(int[] values) {
+        for (int ii = values.length - 1; ii >= 0; ii--) {
```

Comment:
The other reason I did it this way is that this is how the Ruby track implements it.  There is no test metadata for this exercise, so I used the Ruby `example.rb` as a template.